README.md

```diff
@@ -20,7 +20,7 @@ addon | version | summary
 [auth_signup_verify_email](auth_signup_verify_email/) | 9.0.1.0.0 | Force uninvited users to use a good email for signup
 [auth_supplier](auth_supplier/) | 9.0.2.0.0 | Auth Supplier
 [base_custom_info](base_custom_info/) | 9.0.1.0.0 | Add custom field in models
-[base_multi_image](base_multi_image/) | 9.0.1.0.0 | Allow multiple images for database objects
+[base_multi_image](base_multi_image/) | 9.0.1.1.0 | Allow multiple images for database objects
```

Comment:
FYI the section `[//]: # (addons)` is auto-generated you don't need add this change.